### DISCUSSION TITLE
Requesting Community Feedback on the "Header" Section of the Website

### DISCUSSION THREAD

**Discussion by jarmos-san** [developersIndia/developersIndia#143] [General]
Thanks to the efforts of @DevanshBajaj @krishnaar21, we've an iteration of the "Header" section of the community which we believe showcases a stronger connection to the Indian software devs & other members of the community.

But before we start implementing the design aesthetics into actual source code, we invite everyone to share your opinion & feedback to it. So, please make full use of the comment section below & don't forget to adhere to our community guidelines as well!

That said, following are a few iterations of the header section we would like your feedback on:

![AlternativeHomePage](https://user-images.githubusercontent.com/31373860/159174366-76bb94b2-4673-4591-bc1e-52b842d7e7f8.png)
![AlternativeHomePage-1](https://user-images.githubusercontent.com/31373860/159174368-41567c27-1521-46cd-851b-8296f1554714.png)
![AlternativeHomePage-2](https://user-images.githubusercontent.com/31373860/159174370-3a8114ca-1133-46e9-806b-f065559e8ef7.png)
![AlternativeHomePage-3](https://user-images.githubusercontent.com/31373860/159174371-fbf8e24a-aac1-4628-8f33-e027e7cab97e.png)
![AlternativeHomePage-4](https://user-images.githubusercontent.com/31373860/159174372-b1f39f1f-9940-4ea1-a8ee-df80364f5367.png)
![AlternativeHomePage-5](https://user-images.githubusercontent.com/31373860/159174373-101f89a4-eb67-4eb6-9d81-78f51d01847a.png)

**Comment by DevanshBajaj** [comment]
I would like to say that 3rd one seems like a best choice **in my opinion** because it represents the "India" aspect as well while keeping the design minimal and not too much distracting.
Also thanks to @krishnaar21 for improving the hero section.

**Reply by jarmos-san** [reply]
Indeed! I like the 3rd one as well since its minimal & leaves enough room for future additions (or improvements).

**Comment by krishnaar21** [comment]
Hey, Thanks for the opportunity guys! I like the 3rd and 4th.  As Devansh said they reflect India & programming process in them respectively.

**Comment by jarmos-san** [comment]
Looks like the 4th one is more popular among the community.

![image](https://user-images.githubusercontent.com/31373860/159231825-5d63e2cd-6e74-4896-b682-26d72ab36161.png)

**Reply by Animesh-Ghosh** [reply]
4th one for me as well! 😃

**Comment by jarmos-san** [comment]
Looking this thread since a majority of the community has already voted for the 4th version & we need to work on a PR (see #144) as well. For tracking progress on the header section of the website, please see #134
